source/kubernetes/auto-healing.rst

```diff
@@ -17,6 +17,86 @@ self-healing that Kubernetes performs for pods.
+|                    | from the node in the last node-monitor-grace-period      |
+|                    | (default is 40 seconds)                                  |
++--------------------+----------------------------------------------------------+
+| DiskPressure       | True if pressure exists on the disk size--that is, if    |
+|                    | the disk capacity is low; otherwise False                |
++--------------------+----------------------------------------------------------+
+| MemoryPressure     | True if pressure exists on the node memory--that is, if  |
+|                    | the node memory is low; otherwise False                  |
++--------------------+----------------------------------------------------------+
+| DiskPressure       | True if pressure exists on the disk size--that is, if    |
+|                    | the disk capacity is low; otherwise False                |
++--------------------+----------------------------------------------------------+
+| PIDPressure        | True if pressure exists on the disk process--that is, if |
+|                    | the disk process is low; otherwise False                 |
++--------------------+----------------------------------------------------------+
+| NetworkUnavailable | True if the network for the node is not correctly        |
+|                    | configured, otherwise False                              |
++--------------------+----------------------------------------------------------+
+| KernelDeadlock     | True if the kernel has no deadlock otherwise False       |
++--------------------+----------------------------------------------------------+
+		
+To support more conditions monitoring and healing, you can just edit the
+configmap named ``magnum-auto-healer-config`` shown as below to support additional
+node conditions.   							
+
+.. code-block:: yaml
+
+  cluster-name: 718439c2-933c-4288-abc7-c3e6ba617663
+  dry-run: false
+  cloud-provider: openstack
+  repair-delay-after-add: 3m
+  openstack:
+    user-id: 937509608ad344d0b226f5946f64d23b
+    password: "password"
+    auth-url: http://192.168.200.200/identity
+    region: RegionOne
+    project-id: d40141b0d5604fbdabfa65dbe8eceb7a
+  kubernetes:
+      api-host:
+      kubeconfig: /home/feilong/config
+  healthcheck:
+      master:
+        - type: Endpoint
+          params:
+            unhealthy-duration: 3m
+            protocol: HTTPS
+            port: 6443
+            endpoints: ["/healthz"]
+            ok-codes: [200]
+        - type: NodeCondition
+          params:
+            unhealthy-duration: 3m
+            types: ["Ready"]
+            ok-values: ["True"]
+      worker:
+        - type: NodeCondition
+          params:
+            unhealthy-duration: 3m
+            types: ["Ready"]
+            ok-values: ["True"]
+        - type: NodeCondition
+          params:
+            unhealthy-duration: 3m
+            types: ["DiskPressure"]
+            ok-values: ["False"]
+
```

Comment:
This looks good, however I think the section should come after **How does it work?**, otherwise you're getting into low-level details before the reader understands what is happening.

Reply:
Good point.